app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -41,4 +41,42 @@ select.form-control.search_options {
     padding: 5px;
     width: 138px;
   }
+}
+
+// pagination
+.pagination a, .pagination span.current, .pagination span.gap {
+  float: left;
+  padding: 0 15px;
+  line-height: 38px;
+  text-decoration: none;
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.